src/main/java/org/commonjava/util/sidecar/services/ReportService.java

```diff
@@ -35,12 +36,26 @@
 import static org.commonjava.util.sidecar.util.SidecarUtils.getBuildConfigId;
 
 @RegisterForReflection
+@ApplicationScoped
 public class ReportService
 {
     private final Logger logger = LoggerFactory.getLogger( getClass() );
 
-    @Inject
-    TrackedContent trackedContent;
+    public TrackedContent trackedContent;
```

Comment:
Avoid to use public field here, or make it final to be immutable if public is needed.